lib/account/pages/help_page.dart

```diff
@@ -12,12 +14,60 @@ class HelpPage extends StatefulWidget {
 class _HelpPageState extends State<HelpPage> {
   List<Widget> getFaqCards() => [
         FAQCard(
-          question: S.of(context).pageHelpWhatIsMotisQuestion,
-          answer: S.of(context).pageHelpWhatIsMotisAnswer,
+          question: S.of(context).pageHelpQuestion1,
```

Comment:
not sure it's a good idea to remove all information from the translation key. Also will be confusing if we choose to reoder.

Reply:
Actually made this change with reordering in mind, since we just need to reorder this in the language files. Otherwise, you would have to go back and forth between language file and actual file if you want to keep them in the same order